# print("Enter First Number")
# num1=input()
# print("Enter second Number")
# num2=input()
# try:
#
#     print("The sum of a num1 & num2 is:", #Select this statement and press Ctrl+Tab to being the the indentation of Try
#           int(num1)+int(num2))
# except Exception as e:
#     print(e)
# print("This line is very important and you wanna excecute it")
# print("This line is very important and you wanna excecute it")
# print("This line is very important and you wanna excecute it")
# print("This line is very important and you wanna excecute it")
# We use try Exception Handling to handle the program if there is any Error in Your program and you want your program and left a  message for a user and doesn't stop the program.
# We Use Try Exception Handling when we work with the internet connectivity and there is some Error In code or internet connection so this won't stop the program and leave s short message for the user.
# We dissolve the matter and save our program through Try Exception as e

#________________________________________________________________________________________________________________________________
# 22-Aug-2022 to 21-Sep-2022
#Final Taking Course

#Try Except Exception Handling In Python

num1=int(input("Enter First Number:\n"))
num2=int(input("Enter second Number:\n"))
try:

    print("The sum of a num1 & num2 is:", #Select this statement and press Ctrl+Tab to being the the indentation of Try
          int(num1)+int(num2))
except Exception as e:
    print(e)
print("This line is very Important")